tests/test_unittest/test_storage.py

```diff
@@ -790,53 +790,95 @@ def test_view_gpu():
     run_test_view(backend="gtcuda")
 
 
-def test_numpy_patch():
-    storage = gt_store.from_array(
-        np.random.randn(5, 5, 5), default_origin=(1, 1, 1), backend="gtmc"
-    )
-
-    class npsub(np.ndarray):
-        pass
+class TestNumpyPatch:
+    def test_asarray(self):
+        storage = gt_store.from_array(
+            np.random.randn(5, 5, 5), default_origin=(1, 1, 1), backend="gtmc"
+        )
 
-    numpy_array = np.ones((3, 3, 3))
-    matrix = np.ones((3, 3)).view(npsub)
+        class npsub(np.ndarray):
```

Comment:
This is just nitpicking but it could be a good opportunity to rename the `npsub` class to a proper name following Python conventions (e.g. `NDArraySub` or `_NDArraySub`)